esp-hal/src/soc/esp32p4/mod.rs

```diff
@@ -0,0 +1,36 @@
+//! # SOC (System-on-Chip) module (ESP32-P4)
```

Comment:
These docs are also a little bit pointless, could you remove (and remove in the other chip soc/mod.rs please :heart: )